src/screens/MessageScreen.tsx

```diff
@@ -286,6 +315,15 @@ const defaultStyles = createStyles('MessageScreen', (theme) => {
       color: theme.colors.text,
       fontWeight: '600',
     },
+    multiGiftedAvatarView: {
+      flex: 1,
+      flexWrap: 'wrap',
+      maxHeight: 40,
+      marginRight: 50,
+    },
```

Comment:
Any chance we could make it round with a background, similar to iOS?
![IMG_4065](https://github.com/lifeomic/react-native-sdk/assets/2295908/25eec31b-76f4-4a7b-b803-7d2e0bba0667)